cromwell_tools/_cli.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+
+from ._cromwell_api import CromwellAPI
+from ._cromwell_auth import CromwellAuth
+
+
+def parser(arguments=None):
```

Comment:
This could be a refactor for another time, but it would be nice if it look less parsing/code to call the right function with the right arguments. I just glanced at this python library [click](https://github.com/pallets/click) that might be useful! It lets you add a decorator to functions and call them from the command line.

Reply:
This is possible. I don't really like `click` because it takes a lot of simple logic and removes it from the library. For people who aren't familiar with the library or decorators it can be confusing. But, maybe it's time I give it another try :)

Reply:
What I _do_ like is the way Tony's done the CLI here: https://github.com/spacetx/slicedimage/tree/master/slicedimage/cli